docs/framework/react/start/api-routes.md

```diff
@@ -7,21 +7,7 @@ API Routes are a powerful feature of TanStack Start that allow you to create ser
 
 By default, API Routes are defined in your `./app/routes/api` directory of your project and are automatically handled by the TanStack Start server.
 
-> 🧠 This means that by default, your API Routes will be prefixed with `/api` and will be served from the same server as your application. You can customize this base path by changing the `apiBase` in your TanStack Start config.
-
-Topics covered in this guide:
```

Comment:
These are being removed because 1) this doesn't exist in other sections, and 2) there is a sidebar right next to the documentation that has exactly the same links.